pyccel/ast/datatypes.py

```diff
@@ -59,8 +59,31 @@
 )
 
 #==============================================================================
-
-default_precision = {'real': 8, 'int': numpy.dtype(int).alignment, 'integer': numpy.dtype(int).alignment, 'complex': 8, 'bool':4, 'float':8}
+iso_c_binding = {
+    "integer" : {
+        1 : 'C_SIGNED_CHAR',
+	    2 : 'C_SHORT',
+        4 : 'C_INT',
+	    8 : 'C_LONG_LONG',
+        16 : 'C_INT128'}, #no supported yet
+    "real" : {
+        4 : 'C_FLOAT',
+	    8 : 'C_DOUBLE',
+        16 : 'C_LONG_DOUBLE'},
+    "complex" : {
+        4 : 'C_FLOAT_COMPLEX',
+	    8 : 'C_DOUBLE_COMPLEX',
+        16 : 'C_LONG_DOUBLE_COMPLEX'},
+    "logical" : {
+		4 : "C_BOOL"}
+}
```

Comment:
There seems some misalignment here, which makes it hard to read. Could you please improve the formatting a bit?